### DISCUSSION TITLE
Zim and scripting language

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#2329] [Q&A] [answered]
Is it possible to add some scripting language to the Zim-program for mass processing of notes (such as removing extra spaces, separating erroneously combined notes, adding dates to notes, etc.) ?

**Comment by jaap-karssenberg** [comment, marked as answer]
Zim is written in python. So I could argue the whole program is written in
a scripting language. You can use plugins to add functionality.

If you prefer other languages the custom tool mechanism can help

For batch processing, just process the files :)

Op ma 27 feb. 2023 21:47 schreef Nick222 ***@***.***>:
